app/form_models/publishers/job_listing/documents_form.rb

```diff
@@ -38,6 +38,6 @@ def document_presence
     return unless vacancy.include_additional_documents
     return if documents.present?
 
-    errors.add(:documents, :blank) unless vacancy.supporting_documents.any?
+    errors.add(:documents, :blank)
```

Comment:
This was the cause of the original bug, allowing users to submit a form without any attached document.